server/manifest.go

```diff
@@ -1,9 +1,17 @@
 package main
 
+import (
+	"strings"
+
+	"github.com/mattermost/mattermost-server/model"
+)
+
 var manifest = struct {
-	ID      string
-	Version string
+	ID             string
+	Version        string
+	RequiredConfig *model.Config
 }{
-	ID:      "com.mattermost.plugin-starter-template",
-	Version: "0.1.0",
+	ID:             "com.mattermost.plugin-starter-template",
+	Version:        "0.1.0",
+	RequiredConfig: model.ConfigFromJson(strings.NewReader(`{"ServiceSettings":{"EnablePostUsernameOverride":true}}`)),
```

Comment:
I know we're generating this, but I wonder if there's anyway to generate the literal `&model.Config` instead.